view/page/linux.php

```diff
@@ -7,8 +7,7 @@
     <div class="span6">
       <h3>For the Efficient and Lazy</h3>
       <ol>
-        <li>Make a folder called <code>lbry</code> where you want everything to reside.</li>
-        <li>Download and run <a href="https://raw.githubusercontent.com/lbryio/lbry-setup/master/lbry_setup.sh" class="link-primary">this script</a> from that folder.</li>
+        <li><code>(mkdir lbry; cd lbry; wget https://raw.githubusercontent.com/lbryio/lbry-setup/master/lbry_setup.sh; chmod 755 lbry_setup.sh; ./lbry_setup.sh; bin/lbry/lbrycrdd -server -daemon)</code></li>
```

Comment:
@jimmykiselak slightly confused by your comment: doesn't the last command do what you want (run the console)?

@teran-mckinney Can you clarify on copy/pasting without new lines? It pasted fine for me both directly to console and to GEdit on Ubuntu. I may fiddle with styles more as well.